akka-docs-dev/rst/java/code/docs/http/javadsl/HttpClientExampleDocTest.java

```diff
@@ -55,6 +56,7 @@ public void testHostLevelExample() {
         .via(poolClientFlow)
         .runWith(Sink.<Pair<Try<HttpResponse>, Integer>>head(), materializer);
     //#host-level-example
+    assert(responseFuture != null);
```

Comment:
I prefer the annotation, too, these assertions are rather mysterious. If nothing else works, a method "supressUnusedWarning" that just does this assert(x != null) is more readable.